action.yml

```diff
@@ -5,6 +5,12 @@ inputs:
   base_url:
     description: 'Base URL of the Martini instance'
     required: true
+  client_id:
+    description: 'Client ID of the Martini instance'
+    required: false
+  client_secret:
+    description: 'Client Secret of the Martini instance'
+    required: false
```

Comment:
It should have a default value in such case according to [the docs](https://github.com/actions/toolkit/tree/main/packages/core#inputsoutputs)